ModernWallet/Classes/ViewController/Menu/DrawerController.swift

```diff
@@ -27,14 +27,12 @@ class DrawerController: KYDrawerController {
     }
     
     func showPortfolio() {
+        //unubscribe from notifications about application oppening events (Dev note : LMW-581)
+        NotificationCenter.default.removeObserver(self, name: .applicationOpened, object: nil)
```

Comment:
Why do you remove observer here, when you didn't add one? Add/remove observer should happen in the same class (because you use `self` as observer)